File: test8.js
function countSalutes(hallway) {
    let count = 0
    for(let i = 0; i < hallway.length; i++) {
        for(let j = i+1; j < hallway.length; j++) {
            if(hallway[i] === '>' && hallway[j] === '<') {
                count += 2
            }
        }
    }
    return count;
}

countSalutes('<---<--->----<')
countSalutes('>----->-----<--<')

File: test8.py
def count_salutes(hallway):
    count = 0
    for i in range(len(hallway)):
        for j in range(i+1, len(hallway)):
            if hallway[i] == '>' and hallway[j] == '<':
                count += 2
    return count

# Example usage
print(count_salutes('<---<--->----<'))
print(count_salutes('>----->-----<--<'))
